packages/web/src/Global.js

```diff
@@ -38,6 +38,12 @@ const checkHashAndScroll = () => {
     }
 };
 
+export const openNewTab = (url) => {
+    return () => {
```

Comment:
What you do here is calling a function which in itself is calling an anonymous function. Lose the `return () => ...`